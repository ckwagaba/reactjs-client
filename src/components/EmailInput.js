import React, { Component } from 'react';

class EmailInput extends Component {
  render () {
    return (
      <div>
        <input type="email" className="text_input email_input" placeholder={this.props.placeholder} value={this.props.value} onChange={this.props.handleInput} />
      </div>
    );
  }
}

export default EmailInput;
