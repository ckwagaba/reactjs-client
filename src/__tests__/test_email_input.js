import React from 'react';
import { shallow } from 'enzyme';
import EmailInput from '../components/email_input';

describe('EmailInput component', () => {

  const component = <EmailInput />;

  it('should exist', () => {
    expect(component).toMatchSnapshot();
  });

});
