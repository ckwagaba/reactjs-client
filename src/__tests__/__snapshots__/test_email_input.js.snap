// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailInput component should exist 1`] = `<EmailInput />`;
